app/bundles/CampaignBundle/Form/Type/CampaignImportType.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Mautic\CampaignBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @extends AbstractType<mixed>
+ */
+class CampaignImportType extends AbstractType
```

Comment:
Please make sure all new classes are final by default and have the strict types configured so we don't have to deal with it with every review round.
```suggestion
final class CampaignImportType extends AbstractType
```

Reply:
fixed